dotnet/samples/04.ai.a.teamsChefBot/README.md

```diff
@@ -0,0 +1,121 @@
+DAVE: No, not yet, why?
+AI:Cooking is a great way to learn how to develop apps for Microsoft Teams. Just like cooking, developing apps requires you to have the right ingredients and tools, and to follow a set of instructions. It also requires creativity and problem-solving skills. So, let's get started! What would you like to learn first?
+DAVE: Which kind of apps can I build for Microsoft Teams?
+AI: Great question! You can build a variety of apps for Microsoft Teams, including bots, messaging extensions, tabs, and connectors. Bots are like the main course of your app - they provide the core functionality. Messaging extensions are like side dishes - they add extra features to your app. Tabs are like garnishes - they provide additional information or visuals. And connectors are like spices - they allow you to integrate with other services.
+```
+
+Notice that the bot remembered Dave's first message when responding to the second.
+
+</details>
+<details open>
+    <summary><h3>Localization across languages</h3></summary>
+    Because this sample leverages GPT for all its natural language modelling, the user can talk to an AI bot in any language of their choosing. The bot will understand and respond appropriately with no additional code required.
+</details>
+
+## Setting up the sample
+
+All the samples in for the C# .NET SDK can be set up in the same way: You can find the step by step instructions here: [Setup Instructions](../README.md).
+
+## Interacting with the bot
+
+Interacting with the bot is simple - talk to it! You can invoke it by using @ mention and talk to it in plain language.
+
+The bot uses the text-davinci-003 model to chat with Teams users and respond in a polite and respectful manner, staying within the scope of the conversation. This is possible due to the `skprompt.txt` file's contents.
+
+## Deploy to Azure
+
+You can use Teams Toolkit for Visual Studio or CLI to host the bot in Azure. The sample includes Bicep templates in the `/infra` directory which are used by the tools to create resources in Azure.
+
+To configure the Azure resources to have an environment variable for the OpenAI Key:
+
+1. In `./env/.env.dev.user` file, paste your [OpenAI API Key](https://openai.com/api/) to the environment variable `SECRET_OPENAI_KEY=`.
+
+The `SECRET_` prefix is a convention used by Teams Toolkit to mask the value in any logging output and is optional.
+
+Use the "Teams Toolkit" > "Provision in the Cloud...", "Teams Toolkit" > "Deploy to the Cloud" from project right-click menu, or from the CLI with `teamsfx provision` and `teamsfx deploy`. [Visit the documentation](https://learn.microsoft.com/microsoftteams/platform/toolkit/provision) for more info on hosting your app in Azure with Teams Toolkit.
+
+Alternatively, you can learn more about deploying a bot to Azure manually in the [Deploy your bot to Azure](https://aka.ms/azuredeployment) documentation.
+
+## Use Azure OpenAI
```

Comment:
Since this applies to all samples, we should have a similar file in `getting-started` that is linked to from all samples (C# and js). Let's discuss in scrum!

Reply:
Good point. I just got the PR merged. I will create a separate PR to update it after we have the common getting-started part.